webpack.config.js

```diff
@@ -26,16 +16,6 @@ function getDefaultConfiguration() {
     plugins: [
       new webpack.ProvidePlugin({ process: "process/browser" }),
       new webpack.ProvidePlugin({ Buffer: ["buffer", "Buffer"] }),
-      // TODO: Remove with #2273
-      new webpack.NormalModuleReplacementPlugin(/^bn.js$/, (resource) => {
```

Comment:
Is this no longer needed?

Reply:
bn.js was used by elliptic, the previous ECC lib before noble

I believe @ckniffen has ruthlessly hunted down all other uses as well